CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+### 2.31.7
+This release includes:
+* Fluent Bit [1.9.10](https://fluentbit.io/announcements/v1.9.10/)
+* Amazon CloudWatch Logs for Fluent Bit 1.9.1
```

Comment:
cloudwatch plugin version changed in last release, but we didn't update release notes oops...

Reply:
I did update the version files so the release still contains the fix...

Reply:
Will fix release notes and changelog after this new release